### PR TITLE
Update tracker change log documentation [DHIS2-16741]

### PR DESCRIPTION
**Pull request by muilpp**
- Added the docs for the two new change log endpoints (data values and tracked entity attribute values).
- Marked the old endpoints in audit as deprecated and to be removed in v43.

### CODE REVIEW

File: src/developer/web-api/audit.md [hunk 1]
```diff
@@ -34,67 +34,6 @@ Example: Get audits for data element `BOSZApCrBni`, org unit `DiszpKrYNg8` and c
 
     /api/33/audits/dataValue?de=BOSZApCrBni&ou=DiszpKrYNg8&co=TkDhg29x18A
 
-### Tracked entity data value audits { #webapi_tracked_entity_data_value_audits }
```
Comment (teleivo) [inline src/developer/web-api/audit.md:37]: Have we removed these endpoints?

File: src/developer/web-api/tracker.md [hunk 2]
```diff
@@ -2130,6 +2130,66 @@ An example of a json response:
 The response will be the same as the collection endpoint but referring to a single tracked
 entity, although it might have multiple rows for each attribute.
 
+#### Tracked entity attribute value change logs `GET /api/tracker/trackedEntities/{uid}/changeLogs`
+
+This endpoint retrieves change logs for the attributes of a specific tracked entity. It returns a list of all tracked entity attributes that have changed over time for that entity.
```
Comment (teleivo) [inline src/developer/web-api/tracker.md:2136]: Maybe shorten to

This endpoint retrieves change logs for a tracked entity's attribute values, showing all changes over time.

File: src/developer/web-api/tracker.md [hunk 3]
```diff
@@ -2130,6 +2130,66 @@ An example of a json response:
...
+               "dataElement":"ebaJjqltK5N",
+               "currentValue":"0"
+            }
+         }
+      }
+   ]
+}
+```
+
+The change log type can be `CREATE`, `UPDATE`, or `DELETE`.
```
Comment (teleivo) [inline src/developer/web-api/tracker.md:2191]: Might be more readable if you make it a bulleted list like

The change log type can be one of
* `CREATE`: adding the explanation for what each means here
* `UPDATE`: ...
* `DELETE`: ...

File: src/developer/web-api/audit.md (hunk 1 above)
Comment (muilpp) [reply src/developer/web-api/audit.md:37]: Not yet, but that's my next step.

**Review by teleivo — APPROVED** [review]

**Pull request merged**